scripts/nordic-publish.js

```diff
@@ -1,210 +1,8268 @@
-#!/usr/bin/env node
+#!/usr/bin/env ts-node
```

Comment:
I guess we still have to change this to node?

Reply:
You are right. But we cannot change this manually any longer when we generate it during installing the package. :-/ 

I could add a `transform` to the webpack configuration that copies the script, but I think an even easier solution is, if we call the script explicitly with node. This would also mean, that we do not need to set the executable bit on the file any longer. So in `package.json` we would have `"nordic-publish": "node ./dist/nordic-publish.js"`. 

Sounds good? I updated the instructions in `Changelog.md` in 64b98ad.

Reply:
Yeah, my other thought would be to rewrite `ts-node` to `node` in the ts-file, since it probably won't be run directly anymore.